Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/SelectorAction.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	public enum SelectorAction
+	{
+		All,
+		AddShortcut,
+		Copy,
+		Cut,
+		Delete,
+		Define,
+		Lookup,
+		Paste,
+		Replace,
+		Select,
+		SelectAll,
+		Share
+	}
+}
```

Comment:
If more actions are needed to be available, this enum can be edited in the future.